packages/subgraph/subgraph.yaml

```diff
@@ -0,0 +1,24 @@
+specVersion: 0.0.5
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: PoolMetadataRegistry
+    network: goerli
+    source:
+      address: "0xEBfADf723E077c80f6058dC9c9202bB613DE07cf"
```

Comment:
not sure if we will be able to test this contract - get the latest one with @devjoaov